examples/paho-mqtt/README.md

```diff
@@ -0,0 +1,40 @@
+### About
+This application demonstrates the usage of the MQTT paho package in RIOT.
+
+### Setup
+For using this example, two prerequisites have to be fulfilled:
+
+1. You need a running MQTT broker like Mosquitto broker for example. Take a look at
+[Mosquitto Broker](https://mosquitto.org/). Check online any guide that will help you setting up the broker into some port (a).
+For example this one for debian base linux users [How to setup a Mosquitto MQTT Server and receive data](https://www.digitalocean.com/community/questions/how-to-setup-a-mosquitto-mqtt-server-and-receive-data-from-owntracks).
+
+2. Your RIOT node needs to be able to speak to that broker at the same port you set in 1.
+
+### Setting up RIOT `native` on Linux
+- Run the script create.sh in dist/tools
+- Replace the value of *DEFAULT_MQTT_BROKER_IP* with the IPv6 address provide by `ifconfig tapbr0`
+- Change the default *MQTT_PORT* if needed
+- You'll be able to change to broker IP or connection PORT within the `con` shell command if needed.
+When running the example
+
+## Running the example
+
+- To connect to a broker, use the `con` command:
+```
+con <broker ip> <broker port> <clientID> <username passwd> <keepalivetime>
+```
+
+* If no parameter is set, it will connect to *DEFAULT_MQTT_BROKER_IP*:*MQTT_PORT* with user=riot password=riot
+
+  * *clientID*: is the client id you set up on the broker.
+  * *username and passwd*: those set in the broker, check online tutorial to do it regarding chosen broker.
+  * *keepalivetime*: keep alive in second for your client. (Default one is 10 secs)
+
+- To subscribe to a topic, run `sub` with the topic name as parameter and a QoS level between 1 to 3 (optional), e.g.
+```
+sub hello/world [2]
```

Comment:
What does `[2]` mean ? is it a reference ? If it's the QoS level, then just give an example, so it can be copy pasted from the readme (people are lazy).